decidim-assemblies/app/queries/decidim/assemblies/admin/admin_users.rb

```diff
@@ -33,6 +33,12 @@ def query
         def organization_admins
           assembly.organization.admins
         end
+
+        def assembly_admins
```

Comment:
Method says "I'll return `assembly_admins`", but it returns the `ids` for these admins.
I'd make the method return the query for the whole objects
```ruby
ids= Decidim::AssemblyUserRole
            .where(assembly: assembly, role: :admin)
            .pluck(:decidim_user_id)
Decidim::User.where(id: ids)
```

Reply:
I agree that the naming is not the most appropiate, for what returns it should append `_ids` but if mantaining the name as you suggest (return objects) would mean to modify Line:26 (query) to :

```rb
Decidim::User.where(id: organization_admins).or(assembly_admins)
```


This method was taken from `decidim/participatory_processes`, so if changed I would also suggest to change the originary.